README.md

```diff
@@ -743,13 +743,25 @@ cd root
 terragrunt destroy-all
 ```
 
-Finally, to see the currently applied outputs of all of the subfolders, you can use the `output-all` command:
+To see the currently applied outputs of all of the subfolders, you can use the `output-all` command:
 
 ```
 cd root
 terragrunt output-all
 ```
 
+Finally, if you make some changes to your project, you could evaluate the impact by using `plan-all` command:
+
+Note: It is important to realize that you could get errors running `plan-all` if you have dependencies between your projects.
```

Comment:
s/if you have dependencies between your projects/if you have dependencies between your projects and some of those dependencies haven't been applied yet/